devel/astyle/Portfile

```diff
@@ -18,15 +18,13 @@ long_description    Artistic Style is a source code indenter, source code format
                     programming languages.
 
 homepage            http://astyle.sourceforge.net
-master_sites        sourceforge:project/astyle/astyle/astyle%20${version}
-distname            ${name}_${version}_macos
-checksums           rmd160  5aed3c48b59aedf02e247652db6fad597ef9bd63 \
-                    sha256  c4eebbe082eb2cb98f90aafcce3da2daeb774dd092e4cf8b728102fded8d1dcf \
-                    size    187918
+master_sites        sourceforge
```

Comment:
Please revert this change to `master_sites`. See explanation at https://trac.macports.org/wiki/howto/AvoidRedirects.

You'll need to change `${version}` to `${branch}` which you'll have to define first: https://trac.macports.org/wiki/PortfileRecipes#branch

Reply:
Thanks for the explanation! This is corrected now.